README.rdoc

```diff
@@ -27,8 +27,13 @@ Convert country names and codes to a standard.
 
  # Set the default
  NormalizeCountry.to = :alpha3
- NormalizeCountry.convert("Mexico")                 # "MEX"
- NormalizeCountry.convert("United Mexican States")  # "MEX"
+ NormalizeCountry.convert("Mexico")                # "MEX"
+ NormalizeCountry.convert("United Mexican States") # "MEX"
+
+ # Extend a list of default countries
+ NormalizeCountry.extend_countries(:alpha2 => "SU", :iso_name => "Soviet Union", :short => "USSR")
```

Comment:
Hi, sorry, forgot about this... 

2 different names for 2 different purposes but there's just one method that provides the same functionality.